src/scripting/lua_widget_methods.cpp

```diff
@@ -437,19 +436,22 @@ static int intf_add_item_of_type(lua_State* L)
  */
 static int intf_add_dialog_item(lua_State* L)
 {
-
 	gui2::widget* w = &luaW_checkwidget(L, 1);
+	utils::optional<int> insert_pos = lua_check<utils::optional<int>>(L, 2);
```

Comment:
Please document the new parameter, both in the comment just above this function and in the emmylua files. (Maybe there are other changes needed in the latter place too?)